community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/index/AbstractLuceneIndex.java

```diff
@@ -200,9 +220,10 @@ public void flush() throws IOException
 
     public void close() throws IOException
     {
-        IOUtils.closeAll( partitions );
-        partitions.clear();
         open = false;
+        List<AbstractIndexPartition> partitionsCopy = new ArrayList<>( partitions );
```

Comment:
i do not think it's needed since index is already closed at this point and we can operate on partitions itself right?